org-static-blog.el

```diff
@@ -71,49 +73,59 @@ the index, archive, tags, or RSS feed."
   "File name of the blog landing page.
 The index page contains the most recent
 `org-static-blog-index-length` full-text posts."
-  :group 'org-static-blog)
+  :group 'org-static-blog
+  :safe t)
 
 (defcustom org-static-blog-index-length 5
   "Number of articles to include on index page."
-  :group 'org-static-blog)
+  :group 'org-static-blog
+  :safe t)
 
 (defcustom org-static-blog-archive-file "archive.html"
   "File name of the list of all blog posts.
 The archive page lists all posts as headlines."
-  :group 'org-static-blog)
+  :group 'org-static-blog
+  :safe t)
 
 (defcustom org-static-blog-tags-file "tags.html"
   "File name of the list of all blog posts by tag.
 The tags page lists all posts as headlines."
-  :group 'org-static-blog)
+  :group 'org-static-blog
+  :safe t)
 
 (defcustom org-static-blog-enable-tags nil
   "Show tags below posts, and generate tag pages."
-  :group 'org-static-blog)
+  :group 'org-static-blog
+  :safe t)
 
 (defcustom org-static-blog-enable-deprecation-warning t
   "Show deprecation warnings."
   :group 'org-static-blog)
 
 (defcustom org-static-blog-rss-file "rss.xml"
   "File name of the RSS feed."
-  :group 'org-static-blog)
+  :group 'org-static-blog
+  :safe t)
 
 (defcustom org-static-blog-page-header ""
   "HTML to put in the <head> of each page."
-  :group 'org-static-blog)
+  :group 'org-static-blog
+  :safe t)
```

Comment:
What does `safe` do?

Reply:
This says that these variables are safe to set in directory or file locals. In this case it seems safe because we just treat them as strings.

https://www.gnu.org/software/emacs/manual/html_node/emacs/Safe-File-Variables.html